crates/core/src/db/datastore/locking_tx_datastore/tx_state.rs

```diff
@@ -150,4 +150,16 @@ impl TxState {
         };
         Some((tbl, blob_store, delete_tables.entry(table_id).or_default()))
     }
+
+    /// `get_row_count` returns`i64` because its possible to have net negative row count in TxState
```

Comment:
FWIW, I don't think this is necessary yet since row counts are only used in read transactions.